jwk.go

```diff
@@ -49,6 +50,8 @@ type rawJsonWebKey struct {
 	Dp *byteBuffer `json:"dp,omitempty"`
 	Dq *byteBuffer `json:"dq,omitempty"`
 	Qi *byteBuffer `json:"qi,omitempty"`
+	// Certificates
+	X5c *byteBuffer `json:"x5c,omitempty"`
```

Comment:
That is incorrect. The `x5c` field is an array of base64-encoded certificates. You can't use `*byteBuffer` here because it encodes to base64url-encoding, which is _not_ the same as base64.